config/routes.rb

```diff
@@ -16,7 +16,8 @@ def put(*_args); end
     match "/v0/*path", :via => [:post], :to => redirect(:path => "/internal/v1.0/%{path}", :only_path => true)
 
     namespace :v1x0, :controller => 'notify', :path => "v1.0" do
-      post '/notify/:klass/:id', :action => 'notify'
+      post '/notify/approval_request/:id', :action => 'notify_approval_request'
```

Comment:
Should we change this to `:request_id`, it would be more inline with the `order_item` route that uses `:task_id`.